eloop.c

```diff
@@ -219,11 +209,10 @@ static void eloop_sock_table_dispatch(struct eloop_sock_table *table,
 static void eloop_sock_table_destroy(struct eloop_sock_table *table)
 {
 	int rc, tc, sock;
-
-	if (table) {
+	if (table->table) {
```

Comment:
This is definitely a better guard than `table` as only `table->table` is dynamically allocated. 👍